src/core/cpu_patches.cpp

```diff
@@ -218,7 +218,7 @@ static void SaveContext(Xbyak::CodeGenerator& c) {
         c.push(Xbyak::Reg64(reg));
     }
     for (int reg = 0; reg <= 7; reg++) {
-        c.sub(rsp, 32);
+        c.lea(rsp, ptr[rsp - 32]);
```

Comment:
(Fwiw I'd change this to lower rsp once at the start and then emit an offset in the vmovdqu instructions. Should be faster, likely smaller and preserves flags if done with a lea)

Reply:
Yeah that would be nice